src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/http-foundation": "~2.4.9|~2.5,>=2.5.4",
         "symfony/http-kernel": "~2.6",
         "symfony/filesystem": "~2.3",
-        "symfony/routing": "~2.3.25,>2.3.25|~2.6,>2.6.4",
+        "symfony/routing": "~2.3.26|~2.6.5|~2.7",
```

Comment:
Don't we have other similar examples in other components which could be simplified as well?